dev-lang/nim/nim-1.6.6.ebuild

```diff
@@ -46,6 +34,31 @@ nim_use_enable() {
 	use "${1}" && echo "-d:${2}"
 }
 
+# Borrowed from nim-utils.eclass (guru overlay).
+nim_gen_config() {
+	cat > nim.cfg <<- EOF || die "Failed to create Nim config"
+	cc:"gcc"
```

Comment:
I have a feeling this can backfire. 
Is `gcc.exe:"$(tc-getCC)"` not enough?

Reply:
It's only used for choosing what flags to pass AFAIK.